assets/wizards/popups/components/prompt-action-card/index.js

```diff
@@ -23,7 +23,7 @@ const PromptActionCard = props => {
 	const [ categoriesVisibility, setCategoriesVisibility ] = useState( false );
 	const [ popoverVisibility, setPopoverVisibility ] = useState( false );
 
-	const { className, description, prompt = {}, segments } = props;
+	const { className, description, prompt = {}, segments, warning } = props;
 	const { id, edit_link: editLink, title, sitewide_default: sitewideDefault } = prompt;
```

Comment:
Here and in a few other spots, sitewide default setting is still handled. Most notably, overlays are now marked as inactive in the UI:

![image](https://user-images.githubusercontent.com/7383192/106723773-c1f81f80-6607-11eb-96b8-c11a4fac01c9.png)

Reply:
I feel like I've fixed this 3x already but it keeps getting overwritten by other PRs! Fixed again in a9dcd85.